lib/email/notifications.js

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const P = require('../promise')
+const utils = require('./utils/helpers')
+
+const SIX_HOURS = 1000 * 60 * 60 * 6
```

Comment:
nit: Maybe add a hint in variable name on what this does. Maybe, `NEW_UNVERIFIED_ACCCOUNT_BOUNCE_DELETE`? Or maybe a comment above.